.gitignore

```diff
@@ -23,3 +23,4 @@ xcuserdata
 /node_modules
 /platform/ios/benchmark/assets/glyphs/DIN*
 /platform/ios/benchmark/assets/tiles/mapbox.mapbox-terrain-v2,mapbox.mapbox-streets-v6
+token
```

Comment:
Use a relative path instead of a file name so other files that happen to be named “token” don’t get ignored.